README.md

```diff
@@ -1,21 +1,235 @@
+|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:--------------:|
+|                                                                                                                                                                        <img src="src/main/resources/icons/unknown.svg" width="30">                                                                                                                                                                        |    Unknown     |
+|                                                                                                                                                                          <img src="src/main/resources/icons/low.svg" width="30">                                                                                                                                                                          |      Low       |
+|                                                                                                                                                                        <img src="src/main/resources/icons/medium.svg" width="30">                                                                                                                                                                         |     Medium     |
+|                                                                                                                                                                         <img src="src/main/resources/icons/high.svg" width="30">                                                                                                                                                                          |      High      |
+|                                                                                                                                                                       <img src="src/main/resources/icons/critical.svg" width="30">                                                                                                                                                                        |    Critical    |
+| <img src="src/main/resources/icons/unknownnotapplic.svg" height="15" width="15"><img src="src/main/resources/icons/lownotapplic.svg" height="15" width="15"><img src="src/main/resources/icons/mediumnotapplic.svg" height="15" width="15"><img src="src/main/resources/icons/highnotapplic.svg" height="15" width="15"><img src="src/main/resources/icons/criticalnotapplic.svg" height="15" width="15"> | Not Applicable |
```

Comment:
in vscode, the readme also contains 'description' column in this table.
Normal severity (for CI state) is not included in here
'not applicable' is the lowest severity, if we want to place it at the bottom er should remove the text that

Reply:
I removed the description column because it didn't add any extra information.
The "Normal" severity is not shown to users anymore, so I removed it too.
I changed the order from highest to lowest to make it look better.